src/view/DialogInputBox.tsx

```diff
@@ -1,31 +1,57 @@
-import React, {useState} from "react";
+import React, {FormEvent} from "react";
 import {Textarea} from "nav-frontend-skjema";
 import {Hovedknapp} from "nav-frontend-knapper";
+import useFieldState from "../utils/useFieldState";
+import {DialogData} from "../utils/typer";
 
-function sendeNyMelding (props: { tekst: string }) {
-    console.log(props);
+function validerMelding(melding: string): string | null {
+    if (melding.length === 0) {
+        return "Melding må ha innhold.";
+    } else if (!melding.replace(/\s/g, '').length) {
```

Comment:
er det ikke bare å "trimme" stringen for whitespaces og sjekke lengen etterpå?
Mye lettere å lese `str.trim()`

Reply:
ok, det bli kun en test med `tema.trim().length === 0`